custom_components/reflex_calendar/calendar.py

```diff
@@ -20,7 +20,13 @@ def _on_active_start_date_change_spec(e0):
 
 
 def _on_change_spec(date) -> list[rx.Var]:
-    return [rx.Var(f"{date}.toDateString()")]
+    return [
+        rx.cond(
+            rx.Var(f"{date}") == "None",
+            rx.Var(f"{date}.toDateString()"),
+            rx.Var(f"{date}")
+        )
+    ]
```

Comment:
```suggestion
    return [
        rx.cond(
            rx.Var(f"Array.isArray({date})"),
            rx.Var(f"{date}.map(d => d.toDateString())"),
            rx.Var(f"{date}.toDateString()"),
        ),
    ]
```
1. The initial condition is not correct, it will always use the False operand.

2.The call to `toDateString` is necessary to display the correct day.
If we don't use it, the start date will show the day prior to the one selected.